vllm/model_executor/layers/sampler.py

```diff
@@ -836,7 +836,7 @@ def _get_prompt_logprob_if_needed(
     # Find prompt logprobs
     prompt_logprobs: Optional[PromptLogprobs] = None
     if is_prompt and sampling_params.prompt_logprobs is not None:
-        prompt_logprobs = []
+        prompt_logprobs = [None]
```

Comment:
The first token should have `prompt_logprobs=None`

Reply:
I think this is not going to work with chunked prefill because the second chunk won't have None for the first prompt logprob